app/helpers/application_helper.rb

```diff
@@ -334,4 +328,25 @@ def get_thematical_areas
       "cards": thematical_page.children.published
     }
   end
+
+  def get_footer_links
+    def make_links slug_array
+      array = []
+      
+      slug_array.each { |slug|
```

Comment:
Two quick comments here:

1. When it's about a multi-line block, it would be better to use `do ... end` instead of curly brackets for better readability.
2. You can probably use `map` instead of `each` so that the last thing you do inside the block gets returned as an element of the array. In this way you won't need to do an extra assignment to another array but just returning the hash for each element